src/main/java/gift/filter/JwtAuthenticationFilter.java

```diff
@@ -22,31 +23,43 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
 
         String uri = request.getRequestURI();
 
-        // admin 경로만 토큰 검사
-        if (uri.startsWith("/admin/members")) {
+        // admin, wishlist 경로 토큰 검사. 로그인이 되어야 이용 가능한 기능
+        if (uri.startsWith("/admin/members") || uri.startsWith("/api/wishes")) {
```

Comment:
2단계 때 admin 페이지에 토큰을 확인하도록 피드백 드렸었는데요. 
잘 구현해 주셔서 토큰 인증 기능이 위시 리스트에 적용 되었기도 하고, 페이지 접근 때문에 신경 쓸 게 더 생기셨을텐데 서버 기능에 집중하기 위해 페이지에 대한 인증은 제외 부탁드릴게요! 
```suggestion
        if (uri.startsWith("/api/wishes")) {
```